src/printer.js

```diff
@@ -1770,7 +1770,8 @@ function genericPrintNoParens(path, options, print, args) {
       if (
         n.attributes.length === 1 &&
         n.attributes[0].value &&
-        isStringLiteral(n.attributes[0].value)
+        isStringLiteral(n.attributes[0].value) &&
+        !n.attributes[0].comments
```

Comment:
Can the attribute node have `trailingComments`?

Reply:
Not sure but I don't think so.. it has a comment with `trailing: true`, but no `trailingComments`.

Reply:
Actually I don't see `trailingComments` anywhere on the code..